gateway/src/apicast/policy/rate_limit/rate_limit.lua

```diff
@@ -104,7 +83,7 @@ function _M:access()
   local keys = {}
 
   for _, limiter in ipairs(self.limiters) do
-    local lim, initerr = init_limiter(limiter)
+    local lim, initerr = traffic_limiters[limiter.name](limiter)
```

Comment:
I'm not sure about this change.

`resty.limit` calls `assert` when instantiating some limiters. See this for example:
https://github.com/openresty/lua-resty-limit-traffic/blob/master/lib/resty/limit/req.lua#L56

I'd expect that the library returned that kind of errors in the second var of the response, but that's not the case. For that reason, I think that we should catch the error like we did before.

What do you think @mikz  ?